x-pack/plugins/lists/server/services/items/write_lines_to_bulk_list_items.ts

```diff
@@ -59,17 +59,17 @@ export const importListItemsToStream = ({
     let list: ListSchema | null = null;
     readBuffer.on('fileName', async (fileNameEmitted: string) => {
       readBuffer.pause();
-      fileName = fileNameEmitted;
+      fileName = decodeURIComponent(fileNameEmitted);
       if (listId == null) {
         list = await createListIfItDoesNotExist({
-          description: `File uploaded from file system of ${fileNameEmitted}`,
+          description: `File uploaded from file system of ${fileName}`,
```

Comment:
While we're here we should probably add i18n for this string; see [something like this](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/common/components/exceptions/translations.ts) for examples of using values with i18n.